sdks/wasm/package-update/download-packages.ps1

```diff
@@ -76,6 +78,18 @@ git reset --hard $asp_remote_name/$asp_branch_name
 
 ./clean.ps1
 
+<ItemGroup>
```

Comment:
the whitespace here is weird, and what happens when you embed raw xml into a powershell script like this? is it a typo?

Reply:
Oh that is definitely a typo. Sorry for not catching that.